### PR TITLE
This should fix it.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: src/Command/PelagosMigrateDatasetFilesCommand.php [hunk 1]
```diff
@@ -174,7 +174,7 @@ protected function setFile(
 
             $fileName = "$dataStore/$udi/$udi.dat";
 
-            if (!file_exists($fileName) or !$ignoreFileExistCheck) {
+            if (file_exists($fileName) and $ignoreFileExistCheck) {
```
Comment (praneethpr) [inline src/Command/PelagosMigrateDatasetFilesCommand.php:177]: We could probably remove this command, because it is a one time thing

Comment (fightingtexasaggie) [reply src/Command/PelagosMigrateDatasetFilesCommand.php:177]: Yeah, I think that's the cleaner approach too. We can always hack it around for testing.

Comment (mickel1138) [reply src/Command/PelagosMigrateDatasetFilesCommand.php:177]: Can we merge it, it was correct at least once, then we will remove it later in another clean up PR.

**Comment by mickel1138** [comment]
Gonna delete this anyway.

**Pull request closed without merging**